Dockerfile

```diff
@@ -74,10 +65,7 @@ WORKDIR /build/
 ADD . /build/
 
 RUN pub get
-# Run dart2js tests that fail in 2.7
-RUN pub run build_runner test --release -- --preset dart2js --tags=dart-2-7-dart2js-variadic-issues
```

Comment:
Can you reinstate this? I think we should continue running all the tests we can on an SDK that matches our lower bound